examples/sep24/fullFlow.ts

```diff
@@ -0,0 +1,63 @@
+import axios from "axios";
+import readline from "readline";
+
+import {
+  walletSdk,
+  IssuedAssetId,
+  SigningKeypair,
+} from "@stellar/typescript-wallet-sdk";
+
+import { runDeposit, runDepositWatcher, depositDone } from "./deposit";
+import { runWithdraw, runWithdrawWatcher } from "./withdraw";
+import { askQuestion } from "./question";
+
+const wallet = walletSdk.Wallet.TestNet();
+const stellar = wallet.stellar();
+const anchor = wallet.anchor({ homeDomain: "testanchor.stellar.org" });
+const account = stellar.account();
+
+let kp: SigningKeypair;
+
+const runFullFlow = async () => {
+  const asset = new IssuedAssetId(
```

Comment:
Can you use USDC please?

Reply:
👍 https://github.com/stellar/typescript-wallet-sdk/pull/58/commits/1222e91f13542b9f296ea0e89f7b70f3f9c46a71